sdk/RELEASE.md

```diff
@@ -14,6 +14,7 @@
 ## Bug Fixes and Other Changes
 * No longer require KFP client for kfp components build [\#7410](https://github.com/kubeflow/pipelines/pull/7410)
 * Require google-api-core>=1.31.5, >=2.3.2 [#7377](https://github.com/kubeflow/pipelines/pull/7377)
+* Depends on `google-cloud-storage>=2.2.1,<3` [\#7502](https://github.com/kubeflow/pipelines/pull/7502)
```

Comment:
`1.8.12` is already out, so this shouldn't sit under it.  Maybe you can create a new section titled `Current Version (Still in Development)` and put this under it? Or feel free to skip the release not for 1.8 branch, and we can handle it at the release time.

Otherwise LGTM.

Reply:
done